Library/.rubocop.yml

```
@@ -2,11 +2,43 @@
 
 require: ./Homebrew/rubocops.rb
 
+inherit_mode:
+  merge:
+    - Include
+    - Exclude
+
 AllCops:
   TargetRubyVersion: 2.6
   DisplayCopNames: false
   # enable all pending rubocops
   NewCops: enable
+  Exclude:
+    - 'Homebrew/bin/*'
+    - 'Homebrew/vendor/**/*'
+
+Cask/Desc:
+  Description: 'Ensure that the desc stanza conforms to various content and style checks.'
+  Enabled: true
```

```suggestion
Cask:
  Enabled: true
```

Would this be able to replace all those below?

---

I'd rather keep them separate for now, so we can maybe use the `config/default.yml` pattern that `rubocop-rspec` uses to enable its default cops.

---

Cool 👍🏻